app/src/main/java/org/oppia/android/app/home/HomeViewModel.kt

```diff
@@ -76,10 +76,16 @@ class HomeViewModel(
       promotedActivityListSummaryDataProvider,
       PROFILE_AND_PROMOTED_ACTIVITY_COMBINED_PROVIDER_ID
     ) { profile, promotedActivityList ->
-      listOfNotNull(
-        computeWelcomeViewModel(profile),
-        computePromotedActivityListViewModel(promotedActivityList)
-      )
+      if (profile.numberOfLogins > 1) {
+        listOfNotNull(
+          computeWelcomeViewModel(profile),
+          computePromotedActivityListViewModel(promotedActivityList)
+        )
+      } else {
```

Comment:
Is this else case being tested? I didn't see any new tests to verify that the promoted story list doesn't show in this case.

Reply:
New test case added. Please lmk if more tests are needed.